pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -91,6 +91,11 @@
                                     Condition="Exists('$(DotNetHostBinDir)/dotnet$(SymbolFileExtension)')">
       <IsSymbolFile>true</IsSymbolFile>
     </ArchitectureSpecificNativeFile>
+    <ArchitectureSpecificNativeFile Include="$(DotNetHostBinDir)/apphost$(ApplicationFileExtension)" />
```

Comment:
I do not follow the purpose of this change @ramarag. M.N.App does not need to have AppHost within it since the latter is only supposed to be used for standalone scenarios. 

Given that, instead of it being pulled in here, have we considered restoring the ApHost package explicitly, to get AppHost executable, as part of the standalone app publish? Doing this will also enable us to remove the following dependency while maintaining the versioning semantic I was describing to @eerhardt in your other PR:

https://github.com/dotnet/core-setup/blob/master/pkg/projects/Microsoft.NETCore.DotNetHost/Microsoft.NETCore.DotNetHostResolver.pkgproj#L19